apps/davi/src/hooks/Guilds/conversions/useBigNumberToNumber.ts

```diff
@@ -7,8 +7,10 @@ export default function useBigNumberToNumber(
   decimals: number,
   precision: number = 2
 ) {
+  // I don't think we need to memoize this function
+  // TODO: check if this is true

```

Comment:
are you leaving this on purpose or didn't realise?

Reply:
I left that on purpose. If it makes sense, from a performance and a code duplication perspective, we should refactor the places we use the hook. However, since that will change a significant amount of code, I wanted to leave it out of the current PR.